blocks/header/header.css

```diff
@@ -910,6 +910,16 @@ html[dir="rtl"] header nav .nav-tools {
   font-weight: normal;
 }
 
+#search-input:-internal-autofill-selected {
+  background-color: var(--input-background-color) !important;
+  color: black !important;
+}
+
+#search-input:-webkit-autofill {
+  background-color: red !important;
+  color: black !important;
+}
+
```

Comment:
why `!important`?

Reply:
These changes are not working, so removing these @thomsebastin